tools/pd-simulator/simulator/cases/region_split.go

```diff
@@ -29,11 +29,10 @@ func newRegionSplit() *Case {
 	storeNum := getStoreNum()
 	for i := 1; i <= storeNum; i++ {
 		simCase.Stores = append(simCase.Stores, &Store{
-			ID:        uint64(i),
-			Status:    metapb.StoreState_Up,
-			Capacity:  1 * units.TiB,
-			Available: 900 * units.GiB,
-			Version:   "2.1.0",
+			ID:       uint64(i),
+			Status:   metapb.StoreState_Up,
+			Capacity: 1 * units.TiB,
+			Version:  "2.1.0",
```

Comment:
Do we need to change the version in master？plz,see @lhy1024

Reply:
I think we can do it in another pr https://github.com/tikv/pd/issues/5672